core/dbt/compilation.py

```diff
@@ -52,7 +52,9 @@ def print_compile_stats(stats: Dict[NodeType, int]):
         dbt.tracking.track_resource_counts(resource_counts)
 
     # do not include resource types that are not actually defined in the project
-    stat_line = ", ".join([pluralize(ct, t) for t, ct in stats.items() if ct != 0])
+    stat_line = ", ".join(
+        [pluralize(ct, t).replace("_", " ") for t, ct in stats.items() if ct != 0]
```

Comment:
I decided to undo my "fix" from https://github.com/dbt-labs/dbt-core/pull/9465, at least for the visual display in the `FoundStats` and `FinishedRunningStats` events